### ISSUE TITLE
Base with transpancy part is always on top of its opaque sibling.

### ISSUE THREAD

**Issue by JenniferLee520** [red/red#1823]
```Red
RED[]

view [
	canvas: panel [
		circle: base 50x50 loose draw: [ line-width 5 circle 24x24 20 ]
			on-drag-start [
				remove find canvas/pane face
				append canvas/pane face
			] 
		base red 50x50 loose "Drag Me"
			on-drag-start [
				remove find canvas/pane face
				append canvas/pane face
			] 
		base green 50x50 loose "Drag Me"
			on-drag-start [
				remove find canvas/pane face
				append canvas/pane face
			] 
	]
	do [
		circle/color: none
	]
]
```
Dragging object will be appended to the last for making it show on the top. but the circle is always on top. That's strange.

**Comment by JenniferLee520** [comment]
BTW. I am using Windows 10.

**Comment by qtxie** [comment]
On Windows, the `BASE` face will use two different implements(normal window and layered window) internally according to the way to use it.
The reason is we cannot make a normal window be a shaped window or a transparent window, only a [layered window](https://msdn.microsoft.com/en-us/library/ms997507.aspx) can do this. But layered window cannot be a child window, we make it as popup window to simulate it, this is why it is always on top of the normal window.
For this case, one solution is make all the base faces be a layered window. Just specify the alpha chanel in the background color. e.g change `red` to `255.0.0.0`.

**Comment by qtxie** [comment]
Also there is a known issue when you remove the face in `on-drag-start` event, the face will lose mouse capture. @dockimbel Did we fix it? I don't remember. :confused:

**Comment by dockimbel** [comment]
@qtxie IIRC, yes we've fixed it. For this ticket, the reported bug in on Win10, we don't use layered windows on that version of Windows, so it should work, right?

**Comment by qtxie** [comment]
No. we use layered windows on all version of windows now.

**Comment by dockimbel** [comment]
@qtxie Is there any way to put a transparent window behind a non-transparent one in Win10?

**Comment by qtxie** [comment]
Yes. on Win8+, we can make a transparent window as child window, so it works well with a non-transparent.

**Comment by qtxie** [comment]
@dockimbel For mouse capture issue, it's just partial fixed. https://github.com/red/red/issues/1513

**Comment by toomasv** [comment]
Is this connected to the above problem, that when specifying `transparent` for a panel, alpha channel is not recognized   on W10?
```
view [base red at 10x10 panel transparent]
```
shows black panel only, while red `base` is expected to be seen.
(build: 3-Mar-2018)
